acceptance/acceptance_test.go

```diff
@@ -145,99 +145,108 @@ func testAcceptance(t *testing.T, when spec.G, it spec.S) {
 		})
 
 		when("default builder is set", func() {
+
```

Comment:
Minor nitpick: this extra space conflicts with our style conventions

Reply:
Oops! Artifact of c/p. Done.